src/tools/gen/support/c.py

```diff
@@ -13,13 +13,13 @@ def funcname(self, key):
 			raise Exception("invalid keyname " + key)
 
 	def getfuncname(self, key):
-		return "get_"+self.funcname(key)
+		return "elektraGet"+self.funcname(key)
```

Comment:
This change would need a fix in the tests. It might be better to have a new py file which inherits from `c.py`.

Please cut off basename here, or fix specification of your user study example to not contain `Sw`.